app/assets/stylesheets/components/_document-table.scss

```diff
@@ -2,26 +2,45 @@
   table-layout: fixed;
   width: 904px;
 
```

Comment:
I think there is some sort of prettification/formatter applied to the document that made some inadvertant changes. I can revert if its a problem.